packages/patternfly-4/react-table/src/components/Table/Body.js

```diff
@@ -19,15 +19,17 @@ const defaultProps = {
 };
 
 class ContextBody extends React.Component {
-  onRow = (row, props) => {
+  onRow = (row, rowProps) => {
     const { onRowClick } = this.props;
     return {
-      isExpanded: row.isExpanded,
```

Comment:
Do not remove `isExpanded` and `isOpen` props. They are needed for correct rendering of collapsible tables

Reply:
The RowWrapper interface was changed accordingly and it allows for more rich props like the ones used in `inline-edit` package. E.g. 

```js
const propTypes = {
  ...ReactTableRowWrapper.propTypes,
  trRef: PropTypes.func,
  className: PropTypes.string,
  onScroll: PropTypes.func,
  onResize: PropTypes.func,
  row: PropTypes.shape({
    isOpen: PropTypes.bool,
    isExpanded: PropTypes.bool,
    isEditing: PropTypes.bool,
    isTableEditing: PropTypes.bool,
    isLast: PropTypes.bool,
    isFirst: PropTypes.bool,
    editConfig: PropTypes.object
  }),
  rowProps: PropTypes.object
};
```

The `isOpen` and `isExpanded` are still passed and used. Only `isExpanded` prop is actually used though, because `isOpen` was not used before and is just passed to the RowWrapper. 

`isOpen` is used correctly (taken from rowData) in the CollapseColumn

Reply:
to conclude, I am not aware of any breaking changes

Reply:
ok, you are right, I didn't try to combine them - something breaks it

Reply:
Oh, okay. I tried to set it up and it does allow collapsible with inline edit. Good job!

Reply:
When looking at the collapsed column it does not cover this case. You are unable to edit such row. Honestly I don't think it's really required and can be left as future improvement.

Reply:
yes, I will try to solve it somehow (if I can in reasonable timeframe) so at least that use case is passable somehow.

I will also have to add some property to indicate that the editing includes the children and show the css and buttons accordingly. We are looking at this exact use case which will be needed in the future